### DISCUSSION TITLE
Copilot billing is misleading or in error

### DISCUSSION THREAD

**Discussion by frin** [community/community#41757] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

My 60 day trial ended and I was just charged for yearly Copilot (which is the period I selected), but instead of a $100 charge, it charged me "$83.29 ($100/year - prorated for 30 days)". Further down it says "For service through: 2023-01-13" which would be in 1 month from now.

What happened here, what is this prorated amount, and why does it say service through for just 1 month? Will I be billed again in 1 month? What happened to the remaining $16.71? Where are those 30 days coming from? My trial was 60 days, not 30, if it is trying to refer to that, even though I don't see why it should.

I believe there was an error in billing, in both amount, and duration.

**Comment by frin** [comment, marked as answer]
I received a reply from GitHub, I will state the explanations here for anyone else who might experience same billing disasters with Microsoft/GitHub:

1. The 30 days prorate is not actually 30 days, but 60 days for the trial period
2. Service through date has a display error, and should not be for 1 month, but for 10 months (12 months minus prorated 60 days of free trial)
3. The prorated final amount is $100 for full year minus 60 days of trial, so $83.29 for payment

In my case, I will be billed again on 13th October 2023, Which is 10 months from now, or 1 year from when I started free trial.

I will keep an eye on the status through the following month to see if they charge me again or if they update the date. In current state it is not possible to view expiration date. The billing page does not state any date, and the last receipt is still broken.

**Reply by mcotdev** [reply]
I had the same problem this morning. I've subsequently cancelled my subscription.

**Reply by DevinBirtciel** [reply]
I just got the same billing error. GitHub has a pending $100 charge on my credit card. Do I need to try and contact them about this to get the result you got?

**Reply by Loque18** [reply]
BRUH... same, I'm cancelling

**Reply by frin** [reply]
The receipt is still wrong, so they haven't regenerated it, however my Copilot is working fine, and the billing page now properly shows correct date for next payment for 10 months after I started paying for the subscription.

**Reply by sjshaffer** [reply]
I have this too.  When are they going to fix the bug?

**Reply by dallas8000-ops** [reply]
I made my payment on May 14, 2026, and my billing history shows it posted about two weeks earlier. I did not have a past-due balance, and I do not agree to being charged for time I did not use. Please review my billing record, correct the payment period/date, and adjust or credit any incorrect charge.

**Comment by crdcj** [comment]
I am being charged for an amount that I do not know the origin. It seems to me that this is a fraud. Anyway, as the charge comes without any description of the service, it is not possible to know what it is about. I use Github a lot, it is one of the most important tools in the world, but unfortunately its collection part is a real garbage! 
![image](https://user-images.githubusercontent.com/66576393/211189008-297ead23-e0a8-4d87-ac5f-6168ecfe0ad9.png)

**Comment by brisco1016** [comment]
Why was I just Charged $10.00 when I just signed up for the 30day free trial today soI could try it out? I would like a refund since I haven't used it and I was charged for a free 30-day trial. Said specifically that I wouldn't be charged until after the tial.

Shouldn't Github be following a code of conduct to it's supporters as well as in doing and abiding by your own policies when charging or taking money firm ones account?

**Reply by SalvaCN** [reply]
Did you managed to get a refund? i have the same problem now :(

**Reply by brisco1016** [reply]
No I did not. Sorry for the extended absence.

**Comment by aguilerasmiguel** [comment]
same here, I cancelled my copilot account but  anyway  github charge me $10, I tried to contact them but my ticket still open for days without any response

**Reply by techUdayMungalpara** [reply]
i just cancle github copilot account  within  two month still github charge me $10 i just raise complain

**Reply by magnito777** [reply]
same to me why this

**Comment by magnito777** [comment]
No they didn't refun
d

**Comment by gavin-burns-US** [comment]
Please review @gavin-burns-US pro+ subscription removal - full payment yesterday - very frequent user and preview tester????

> [fdddea2cb418a7aa750c8a2575aa3232956c8115.png.pdf](https://github.com/user-attachments/files/27807559/fdddea2cb418a7aa750c8a2575aa3232956c8115.png.pdf)

@copilot .... <img alt="CleanShot 2026-05-15 at 11 53 24@2x" width="2000" height="808" src="https://private-user-images.githubusercontent.com/118209746/593217160-18571e12-8175-4bb3-885d-c6e0f8676b1c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Nzg4NjA3MjUsIm5iZiI6MTc3ODg2MDQyNSwicGF0aCI6Ii8xMTgyMDk3NDYvNTkzMjE3MTYwLTE4NTcxZTEyLTgxNzUtNGJiMy04ODVkLWM2ZTBmODY3NmIxYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNTE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDUxNVQxNTUzNDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGYxMzdkNWQ1MzVkZjk0NWZmYzNmZDdkODdmYjYwYTg2NTMyNWZjYjU4M2UyY2NhNzM4NmI4OWYwNDUzNGIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.Wsz3KjYpZi97Q5CsaELWyGygaMEKSWK-MK_xW9BxLdg">

[fdddea2cb418a7aa750c8a2575aa3232956c8115.png.pdf](https://github.com/user-attachments/files/27807728/fdddea2cb418a7aa750c8a2575aa3232956c8115.png.pdf)

> GitHub receipt
>
> * Date: 2026-05-14 12:31 PM PDT
> * Account billed: `gavin-burns-US`
> * Transaction ID: `9XW37303LW6143638`
> * Charged to: PayPal
> * GitHub Copilot Usage: `$143.74 USD` for Apr 1, 2026 - Apr 30, 2026
> * GitHub Copilot Pro+ - month: `$39.00 USD` for May 14, 2026 - Jun 13, 2026
> * Total: `$143.74 USD*`

Sophia Hayes (GitHub Support) replied on May 14, 2026, 8:22 PM UTC with the plan update details, inclu
<img width="2118" height="2144" alt="CleanShot 2026-05-15 at 13 54 42@2x" src="https://github.com/user-attachments/assets/a0f91304-abbe-4396-bea5-60b13547d401" />
ding paused signups, usage limits, and Opus model availability changes.
<img width="2218" height="2840" alt="CleanShot 2026-05-15 at 14 16 33@2x" src="https://github.com/user-attachments/assets/0d8d7fd8-23ec-46bd-bacc-44427e1b9cf2" />


The ticket was deleted after that comment:

> Sophia Hayes (GitHub Support)May 14, 2026, 8:22 PM UTCHi there Gavin — thanks for reaching out.On April 20, 2026 we announced updates to GitHub Copilot Pro and Pro+ plans. These changes may affect your ability to sign up or the model options you see, and may introduce usage limits for some features.What changed (high level)Copilot Pro and Pro+ signups are temporarily paused in some cases.Some experiences now include rate limits/usage limits.Opus models are no longer available in Copilot Pro.Opus 4.5 and Opus 4.6 have been removed from Copilot Pro+. Opus 4.7 remains available in Copilot Pro+Where to find the official detailsBlog post: https://github.blog/news-insights/company-news/changes-to-github-copilot-individual-plans/Changelog: https://github.blog/changelog/2026-04-20-changes-to-github-copilot-plans-for-individuals/GitHub Community discussion (updates & Q&A): https://github.com/orgs/community/discussions/192963What you can do nextIf you're currently using Copilot Pro or Pro+: your access should continue, but you may notice updated model availability and/or usage limits depending on the feature.If you're trying to sign up for Copilot Pro or Pro+ and can't complete signup: this is expected while signups are paused. Please follow the Community discussion above for the latest updates and timing.If you need Copilot for work or ongoing team use: consider Copilot Business or Copilot Enterprise, which are managed by an organization and may be a better fit for team environments.
-- | --
